### DISCUSSION TITLE
Device media query order issue

### DISCUSSION THREAD

**Discussion by adrianrguez** [GrapesJS/grapesjs#4163] [Q&A]
Hi!

I have a issue: https://youtu.be/el9TLDS2KBk

I have this Devices config: 

```
deviceManager: {
		devices: [{
			name: 'Mobile',
			width: '320',
			widthMedia: '1',
			priority: 3
		},{
			name: 'Tablet',
			width: '768',
			widthMedia: '500',
			priority: 2
		}, {
			name: 'Desktop',
			width: '',
			widthMedia:'1024',
			priority: 1
		}]
	},
```

And I set up a css property in each device... (like you can see in youtube video).

but, why editor not respect the priority in canvas but does in real html view?

I hope you can help me.
Thanks!

**Comment by artf** [comment]
Hi @adrianrguez if you believe there is an issue in the core, please open a proper issue bug (with reproducible demo).
